Assets/MRTK/Core/Utilities/Editor/XRSettingsUtilities.cs

```diff
@@ -7,64 +7,179 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.XR;
+using System.Linq;
+#if XR_MANAGEMENT_ENABLED
+using UnityEngine.XR.Management;
+using UnityEditor.XR.Management;
+#endif // XR_MANAGEMENT_ENABLED
 #endif // UNITY_2019_3_OR_NEWER
 
 namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
 {
     /// <summary>
-    /// Utilities that abstract XR settings functionality so that the MRTK need not know which
-    /// implementation is being used.
+    /// Utilities that abstract XR settings functionality
     /// </summary>
     public static class XRSettingsUtilities
     {
-#if UNITY_2019_3_OR_NEWER
-        static XRSettingsUtilities()
+        /// <summary>
+        /// Is either LegacyXR pipeline or XRSDK pipeline enabled?
+        /// </summary>
+        public static bool XREnabled
         {
-            // Called when packages are installed or uninstalled
-            EditorApplication.projectChanged += EditorApplication_projectChanged;
+            get
+            {
+                return XRSDKEnabled || LegacyXREnabled;
+            }
+        }
+
+
+        /// <summary>
+        /// Checks whether the XRSDK pipeline is properly set up to enable XR.
+        /// </summary>
+        /// <remarks>
+        /// <para>Returns true if using the Unity OpenXR plugin along with the Microsoft plugin or using the built-in plugins.
+        /// Returns false if the only enabled plugin is Unity OpenXR and the Microsoft one is not present, or there is no enabled plugin.</para>
+        /// </remarks>
+        public static bool XRSDKEnabled
```

Comment:
I really like adding all of these utility variables! It'll probably need to get changed after #9681 as a heads up